openstudiocore/CMakeLists.txt

```diff
@@ -1045,30 +1041,22 @@ if(NOT UNIX)
   endif()
 endif()
 
-if(PACKAGE_RADIANCE)
-  set(RADIANCE_LOCATION "" CACHE PATH "Location of Radiance to Package")
-
-  if(APPLE)
-    install(DIRECTORY "${RADIANCE_LOCATION}" DESTINATION "sharedresources" USE_SOURCE_PERMISSIONS)
-  elseif(MSVC)
-    install(DIRECTORY "${RADIANCE_LOCATION}" DESTINATION "share/openstudio" USE_SOURCE_PERMISSIONS)
-  else()
-    install(DIRECTORY "${RADIANCE_LOCATION}" DESTINATION "share/openstudio-${OPENSTUDIO_VERSION}" USE_SOURCE_PERMISSIONS)
-  endif()
+# Package Radiance
+set(RADIANCE_LOCATION "" CACHE PATH "Location of Radiance to Package")

```

Comment:
Nah, it's the same as `ENERGYPLUS_DIR`.  All of the logic is handled at the top level, so that variable is already set in the CMake cache by the time it gets to this line.